### PR TITLE
Added tests for Request utils

### PR DESCRIPTION
**Pull request by abinav62**
Added tests to make sure output is proper on success

### CODE REVIEW

File: tests/test_requests.py [hunk 1]
```diff
@@ -895,3 +895,61 @@ def test_display_challenge_details_for_request_exception(self):
...
+        expected = "Your participant team TeamTest was successfully created."
+        expected = "{}{}".format(user_prompt_text, expected)
+        assert response == expected
+
+    @responses.activate
+    def test_participate_in_a_challenge_success(self):
+        runner = CliRunner()
+        result = runner.invoke(challenge, ["2", "participate", "3"])
+        response = result.output
+        expected = "Your team id {} is now participating in this challenge.".format("2")
```
Comment (Ram81) [inline tests/test_requests.py]: @abinav62 instead of hardcoding `2` here can you create a class variable like `self.team_id = 2` and use it here

**Comment by pushkalkatara** [comment]
@abinav62 The build is failing.

File: tests/test_requests.py [hunk 2]
```diff
@@ -895,3 +895,64 @@ def test_display_challenge_details_for_request_exception(self):
...
+        expected = "Your participant team TeamTest was successfully created."
+        expected = "{}{}".format(user_prompt_text, expected)
+        assert response == expected
+
+    @responses.activate
+    def test_participate_in_a_challenge_success(self):
+        runner = CliRunner()
+        result = runner.invoke(challenge, [self.challenge_id, "participate", self.team_id])
+        response = result.output
+        expected = "Your team id {} is now participating in this challenge.".format("2")
```
Comment (pushkalkatara) [inline tests/test_requests.py]: Please change it with self.team_id as you have declared above.

File: tests/test_requests.py [hunk 3]
```diff
@@ -895,3 +895,64 @@ def test_display_challenge_details_for_request_exception(self):
...
+            status=201)
+
+        responses.add(
+            responses.POST,
+            url.format(API_HOST_URL, URLS.host_team_list.value),
+            status=201)
+
+        responses.add(
+            responses.POST,
+            url.format(API_HOST_URL, URLS.participate_in_a_challenge.format("2", "3")),
```
Comment (pushkalkatara) [inline tests/test_requests.py]: please change 2 and 3 with self.team_id and self.challenge_id.

**Comment by abinav62** [comment]
When I try to test my build on my machine, it says it cannot find .base from the imports. What is .base?

**Comment by abinav62** [comment]
Tests no longer break.

**Comment by pushkalkatara** [comment]
LGTM :+1:

**Pull request merged**